### ISSUE TITLE
Create updated Headings Violations script

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#11900]
## Description
We previously had an export that was used to find the pages on the website with heading violations.  However, we discovered the script that was used to produce this export failing to report a lot of problems because of how it was written (see [comments ](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/11701#issuecomment-1344454340)in the previous ticket).

The script will need to be rewritten in order get an accurate report of all violations.

For future purposes, the new script should also include getting the Last Editor of the page.  This was previously handled manually but as we are anticipating more violations, it will be more efficient to handle this within the script.

### Additional Context
Cases that pass using the current script that should not:
```
<h1></h1>
<h2></h2>
<h4></h4>
```
```
<h1></h1> 
<h3></h3>
<h2></h2>
<h3></h3>
```
```
<h1></h1>
<h5></h5>
<h2></h2>
<h6></h6>

```
### Examples pages that should be in the export

- https://www.va.gov/tuscaloosa-health-care/programs/resource-guides-and-new-patient-information/
- https://www.va.gov/decision-reviews/higher-level-review/
- https://www.va.gov/records/download-va-letters/

## Acceptance Criteria
- [ ] Rewrite heading violations script
- [ ] Ensure the new script will report on the above mentioned cases 
- [ ] Ensure the script gets the Last Editor of each page
- [ ] Generate export of heading violations and editors

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [x] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by ndouglas** [comment]
```
root@php:/var/lib/tugboat/web# wc -l heading_order_violations.txt 
16797 heading_order_violations.txt
```

Looks like there are slightly under 17,000 of these at the moment 😩

**Comment by laflannery** [comment]
@ndouglas I'm just going through and cleaning some things up - is this still in progress? Definitely fine if it is but also I know you opened a few, possibly more specific issues. From what I've seen you have fixed the script itself and now you are just working on the reporting and display?

**Comment by ndouglas** [comment]
I think this specific issue can be closed.  

I have this PR open https://github.com/department-of-veterans-affairs/va.gov-cms/pull/12692 to complete reporting the metric data to DataDog.

I haven't really come up with a good solution for making that `report_heading_violations.html` more discoverable, though.

Anything I'm missing?

**Comment by laflannery** [comment]
I don't think so - I feel like on a day -to-day situation, the updates folks are making they shouldn't necessarily need to check this as part of their process. This is more for me and for people/teams when we are on purpose trying to do things to help the situation - in which case just going to the file directly is not that big of a deal, in my opinion.
